### ISSUE TITLE
Missing tag id to remove

### ISSUE THREAD

**Issue by macortesn** [dazuaz/ValiuTagAmount#6]
Remove tag does not work because the request is not being made correctly.
https://github.com/dazuaz/ValiuTagAmount/blob/94a0a46695422acbc851af30a936957efe92c96a/components/TagListApi.tsx#L33

**Comment by dazuaz** [comment]
Yes, must have missed adding it back after a refactor, will update soon.
